.github/workflows/sync_trackers.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         cp .github/workflows/githubcom_jira_usermap.toml githubcom_jira_usermap.toml
         sed -i -e "s|\$secrets.gitlabcom_token|$GITLABCOM_TOKEN|g" config.toml
         cp .github/workflows/gitlabcom_jira_usermap.toml gitlabcom_jira_usermap.toml
+        sed -i -e "s|\$secrets.codebergorg_token|$CODEBERGORG_TOKEN|g" config.toml
```

Comment:
Do I need to add the token? Or did you already do that?

Reply:
I didn’t yet, just followed the general pattern. I think that we might get by without a token at this point, just not sure if we would run into any API limits. When there are private issues, we obviously need to use a token so the script can access them.